src/index.html

```diff
@@ -15,8 +15,92 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="links">
+            <a
+              href="#"
+              class="link link-1 is-active"
```

Comment:
Why do you need `link-1 ` class?
You can use `:first-child` for that - https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Selectors/:first-child